File: app/src/main/java/com/gitlqr/litearouter/demo/MyApplication.java
package com.gitlqr.litearouter.demo;

import android.app.Application;

import com.gitlqr.litearouter.launcher.LiteARouter;

public class MyApplication extends Application {
    @Override
    public void onCreate() {
        super.onCreate();
        LiteARouter.openDebug();
        LiteARouter.openLog();
        LiteARouter.init(this);
    }
}

File: AndroidXDemo/app/src/main/java/com/gitlqr/litearouter/androidxdemo/MyApplication.kt
package com.gitlqr.litearouter.androidxdemo

import android.app.Application
import com.gitlqr.litearouter.launcher.LiteARouter

class MyApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        LiteARouter.openDebug()
        LiteARouter.openLog()
        LiteARouter.init(this)
    }
}